tsconfig.json

```
@@ -0,0 +1,14 @@
+{
+  "include": ["**/**.js", "*.js"],
```

```suggestion
  "include": ["**/*.js"],
```

This should be enough: a) double asterisk doesn’t do anything for files, b) `**/` also means zero folders, so the second case isn’t needed